src/main/java/org/opensearch/geospatial/geojson/GeoJSONFactory.java

```diff
@@ -0,0 +1,36 @@
+/*
+ * Copyright OpenSearch Contributors
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+package org.opensearch.geospatial.geojson;
+
+
+import java.util.Map;
+
+public class GeoJSONFactory {
```

Comment:
Should this be FeatureFactory if it produces a Feature?

Reply:
Ack